### DISCUSSION TITLE
Recursive function python bug

### DISCUSSION THREAD

**Discussion by demian2435** [codewars/codewars.com#2837] [General]
```
[redacted]
```

# this simple recursive function bug on codewars, XD!!!
on kata **The observed PIN**

**Comment by demian2435** [comment]
```
[redacted]
```
this is how it works on codewars

**Comment by Blind4Basics** [comment]
Hi,

Please, open a thread in the disourse page of the kata. Discussions here have generic purpose, and aren't (generally) specific to one kata. 
For your current matter: your code is incorrect as soon as more than one call is done to your function `get_pins`. Hint: `DON'T USE GLOBAL VARIABLES`. Yeah, you saw the problem. It's definitely not codewars problem, it's "yours", not designing appropriately your solution.

Cheers

note: and don't post solutions publicly. I'll edit your messages to remove the code

**Reply by demian2435** [reply]
thank you!
